packaging/language/easy_install.py

```diff
@@ -137,6 +142,8 @@ def main():
     executable = module.params['executable']
     site_packages = module.params['virtualenv_site_packages']
     virtualenv_command = module.params['virtualenv_command']
+    executable_arguments = []
+    module.params['state'] is 'latest' and executable_arguments.append('--upgrade')
```

Comment:
more readable to do this as a regular if-then rather than a one-liner.

When comparing to a string, use == rather than is.